incubator/kong/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+{{- if and (not .Values.postgres) (not .Values.cassandra) }}
```

Comment:
Can you please add some more info for users on what they can/should do next to interact with Kong?